packages/@react-spectrum/s2/src/Popover.tsx

```diff
@@ -242,30 +241,37 @@ const dialogStyle = style({
   position: 'relative',
   width: 'full',
   maxSize: 'inherit'
-}, getAllowedOverrides({height: true}));
+});
 
 /**
  * A popover is an overlay element positioned relative to a trigger.
  */
-export const Popover = forwardRef(function Popover(props: PopoverDialogProps, ref: DOMRef) {
+export const Popover = forwardRef(function Popover(props: PopoverDialogProps, ref: DOMRef<HTMLDivElement>) {
+  [props, ref] = useSpectrumContextProps(props, ref, PopoverContext);
   let domRef = useDOMRef(ref);
-  const {triggerRef, isOpen, onOpenChange, ...otherProps} = props;
-
+  let {
+    children,
+    disableInnerDiv,
```

Comment:
super gross, maybe we just export PopoverBase as something like CustomPopover to mirror CustomDialog